src/pages/index.js

```diff
@@ -3,95 +3,157 @@ import Card from "/src/components/Card.js";
 import FormValidator from "/src/components/FormValidator.js";
 import Section from "/src/components/Section.js";
 import UserInfo from "/src/components/UserInfo.js";
+import PopupWithConfirmation from "/src/components/PopupWithConfirmation.js";
 import PopupWithForm from "/src/components/PopupWithForm.js";
```

Comment:
Please, delete `evt.target.reset();`  from `PopupWithForm.js` : you need to **reset** the form only in block `then` not to do that immediately after `submit` because when there is an error the popup will still be opened and the user can try to send the data again to the server without having to type the data into the inputs once more